File: challenging/15_pandas/pandas_math.py
import pandas as pd

def pandas_operation(ds1, ds2):
    add = ds1 + ds2

    sub = ds1 - ds2

    mul = ds1 * ds2

    div = ds1 / ds2

    return list(map(lambda x: x.tolist(), [add, sub, mul, div]))
    
ds1 = pd.Series([2, 4, 6, 8, 10])
ds2 = pd.Series([1, 3, 5, 7, 9])
print(pandas_operation(ds1, ds2))

File: challenging/15_pandas/pandas_math.js
function pandas_operation(ds1, ds2) {
    let add = ds1 + ds2;

    let sub = ds1 - ds2;

    let mul = ds1 * ds2;

    let div = ds1 / ds2;

    return [add, sub, mul, div];
}

let ds1 = [2, 4, 6, 8, 10];
let ds2 = [1, 3, 5, 7, 9];
console.log(pandas_operation(ds1, ds2));

module.exports = pandas_operation;